src/components/AnalyticsFilterMenuBar/AnalyticsFilterMenuBar.tsx

```diff
@@ -1,17 +1,42 @@
-import React, { useState } from 'react';
+import React, { ChangeEventHandler, useCallback, useRef, useState } from 'react';
 import { HireStream } from '@prisma/client';
-import { FormTitle, Modal, ModalContent, ModalHeader } from '@taskany/bricks';
+import {
+    FilterPopup,
+    FiltersMenuContainer,
+    FiltersMenuItem,
+    FiltersPanelContainer,
+    FiltersPanelContent,
+    FilterTabLabel,
+    FormTitle,
+    Modal,
+    ModalContent,
+    ModalHeader,
+    nullable,
+    Tab,
+} from '@taskany/bricks';
+import { Button, RadioControl, RadioGroup } from '@taskany/bricks/harmony';
 
-import { FiltersPanel } from '../FiltersPanel/FiltersPanel';
 import { CustomPeriodForm } from '../CustomPeriodForm/CustomPeriodForm';
 import { useAnalyticsFilterUrlParams } from '../../hooks/useAnalyticsFilterUrlParams';
+import { Filter } from '../Filter';
+import { AnalyticsFilterApplied } from '../AnalyticsFilterApplied/AnalyticsFilterApplied';
 
-import { tr } from './AnalyticsFilterMenuBar.i18n';
+import { I18nKey, tr } from './AnalyticsFilterMenuBar.i18n';
 
 interface AnalyticsFilterMenuBarPropsType {
     hireStreams?: HireStream[];
 }
 
+type PeriodMapKey = 'Custom period' | 'Year' | 'Weak' | 'Month' | 'Quarter';
+
+const periodMap: Record<PeriodMapKey, string> = {
```

Comment:
tr should be called inside component render, otherwise dynamic langauge change won't work